packages/web/src/scss/components/Offcanvas/README.md

```diff
@@ -0,0 +1,79 @@
+# Offcanvas
+
+Offcanvas is a sidebar component that can be toggled via JavaScript to appear from the left, right, or bottom edge of the viewport.
+Buttons or anchors are used as triggers that are attached to specific elements you toggle, and data attributes are used to invoke JavaScript plugin.
+
+## JavaScript Plugin
```

Comment:
Link to the main readme for how to include JavaScript plugins?

Reply:
Linked.